### DISCUSSION TITLE
SkyWalking performance profiling has been without data for a long time. Is there any way to troubleshoot this? Version 8.81

### DISCUSSION THREAD

**Discussion by 65725738** [apache/skywalking#9667] [Q&A] [answered]
SkyWalking performance profiling has been without data for a long time. Is there any way to troubleshoot this? Version 8.81
There are many people on the Internet who have similar questions and can't find the answer.

**Comment by wu-sheng** [comment]
Most likely, you choose the wrong endpoint name. You may put an URI, but not an endpoint name in the profiling task.
Remember, we asked for an endpoint, then, copy/paste an endpoint in the SkyWalking, don't just guess and put what you thought it is an endpoint.

Also, we have ended 8.x, https://skywalking.apache.org/events/deprecate-v8/. 9.2.0 was released for days, we only follow the 9.x relative discussion. 9.2.0 has provided endpoint selector, rather than an input to avoid your random input.

**Comment by 65725738** [comment, marked as answer]
I must have chosen the endpoint instead of the URL. And I installed in 2 environments, a Docker, a normal virtual machine. No data on either of them..
![微信图片_20220922155321](https://user-images.githubusercontent.com/12229436/191690161-c565a05b-6f7f-4f8f-a6f6-bd79235a125a.jpg)
![微信图片_20220922155425](https://user-images.githubusercontent.com/12229436/191690181-877a8675-6065-424a-a18b-c6e19f107f69.jpg)

I just touched Skywalking and wanted to choose 9.2, but I didn't see the agent of 9.2. I don't know what the situation is, look at the information on the Internet, 8.7 8.8 is more. So I chose 8.8

**Reply by wu-sheng** [reply]
You didn't read the docs again, https://skywalking.apache.org/docs/main/next/en/setup/service-agent/agent-compatibility/

We never has 9.x Java agent.

And you didn't put `GET:` `POST:` as the endpoint, again, you are guessing, not following what we provided.

**Comment by 65725738** [comment]
Sorry, it's my fault, I didn't think to add POST. The reason is that I didn't add anything on the Internet. Don't you need to add anything before 8.5? I was misled.

**Reply by wu-sheng** [reply]
The endpoint is an endpoint, which is determined by the agent, no the backend change.
